content/en/docs/setup/production-environment/windows/intro-windows-in-kubernetes.md

```diff
@@ -233,21 +233,31 @@ Overlay (VXLAN) networks on Windows do not support dual-stack networking today.
 
 ### Limitations
 
-#### Control Plane
-
 Windows is only supported as a worker node in the Kubernetes architecture and component matrix. This means that a Kubernetes cluster must always include Linux master nodes, zero or more Linux worker nodes, and zero or more Windows worker nodes.
 
-#### Compute {compute-limitations}
-
-##### Resource management and process isolation
+#### Resource Handling
 
  Linux cgroups are used as a pod boundary for resource controls in Linux. Containers are created within that boundary for network, process and file system isolation. The cgroups APIs can be used to gather cpu/io/memory stats. In contrast, Windows uses a Job object per container with a system namespace filter to contain all processes in a container and provide logical isolation from the host. There is no way to run a Windows container without the namespace filtering in place. This means that system privileges cannot be asserted in the context of the host, and thus privileged containers are not available on Windows. Containers cannot assume an identity from the host because the Security Account Manager (SAM) is separate.
 
-##### Operating System Restrictions
+#### Resource Reservations
```

Comment:
Kubernetes has _requests_ and _limits_. Which of these are we discussing?

Reply:
The resource I am talking about are CPU, memory on the node, usually the values we reserve for resources are subtracted from allocatable on the node.